tests/API/Management/BaseManagementTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Auth0\Tests\API\Management;
+
+use Auth0\SDK\API\Management;
+use Auth0\Tests\API\ApiTests;
+use GuzzleHttp\Psr7\Response;
+use Http\Client\Common\HttpMethodsClient;
+use Http\Client\HttpClient;
+use Http\Discovery\MessageFactoryDiscovery;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+abstract class BaseManagementTest extends ApiTests
+{
+    /**
+     * @param HttpClient $httpClient
+     *
+     * @return Management
+     */
+    protected function getManagementApi($httpClient)
+    {
+        return new Management(
+            'token', 'domain.com', new HttpMethodsClient($httpClient, MessageFactoryDiscovery::find())
+        );
+    }
+
+    /**
+     * @return Response
+     */
+    protected function createResponse($body = null, $httpStatus = 200, $headers = [])
```

Comment:
It can be moved later when needed, I don't care so much about tests as internal.